### DISCUSSION TITLE
Heart icon in log section

### DISCUSSION THREAD

**Discussion by LettaLet** [2Abendsegler/GClh#2190] [Ideas]
Hi,
Is it possible to put a heart in the log section if someone has favorited the cache. Perhaps at position 1 or 2?

![Heart_log](https://user-images.githubusercontent.com/101531249/190855660-eb4f66f1-f6cf-48b6-af92-fecf3646eeef.JPG)

**Comment by capoaira** [comment]
We tried it some time ago in #1054, but sadly never find a technical good working solution
